app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/LibraryFragment.java

```diff
@@ -20,6 +20,8 @@
 
 public class LibraryFragment extends Fragment {
 
+    public static final String BOOK_ACTION_KEY = "com.cmput301f20t21.bookfriends.BOOK_ACTION";
```

Comment:
we might need to put this into the BOOK_ACTION enum file. Or, with
```java
BOOK_ACTION.class.getName()
``` 
you get the same result

Reply:
sorry didn't notice trung's comment. I'm good with both solution.